apps/openmw/mwmechanics/aipackage.cpp

```diff
@@ -117,22 +122,26 @@ bool MWMechanics::AiPackage::pathTo(const MWWorld::Ptr& actor, const ESM::Pathgr
         {
             if (wasShortcutting || doesPathNeedRecalc(dest, actor.getCell())) // if need to rebuild path
```

Comment:
doesPathNeedRecalc() checks only the old pathfinding system state, but we did not try to use Detour (buildOptimalPath) yet.

Reply:
It was so at some point, now it is different.